azext_edge/edge/_help.py

```diff
@@ -851,101 +795,122 @@ def load_iotops_help():
         examples:
         - name: Delete an asset.
           text: >
-            az iot ops asset delete --name {asset_name} -g {resource_group}
+            az iot ops asset delete --name myasset -g myresourcegroup
```

Comment:
question: do we only need instance name during asset creation, but not update or deletion?

Reply:
yes - we just need it for creation for the custom location. after we can find it easily